### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by sf-jeff-lim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,209 +1,177 @@
...
+                                        </MainPanel>
+                                </Route>
+                        </Route>
+                        ...
+                </Router>
+        </SendBirdProvider>
 }
 ```
 
-### Color Set List
```
Comment (sravan-s) [inline README.md:108]: @sf-jeff-lim we need the color set list here

File: README.md [hunk 2]
```diff
@@ -1,209 +1,177 @@
...
+This section shows the prerequisites you need to check to use Sendbird UIKit for React.
+
+### Requirements 
+
+The minimum requirements for Sendbird UIKit for React are:
+
+- `React 16.8.0+`
+- `React DOM 16.8.0+`
+- `Sendbird Chat SDK for JavaScript 3.0.115+`
+- `moment 2.24.0+`
```
Comment (sravan-s) [inline README.md:48]: This part changed in last update see - https://github.com/sendbird/SendBird-Docs/pull/1807/files#diff-0b117f748099111cea457d476d04263e2dd0740f7b457f097246a9b8b1e29075L27-R30

File: README.md [hunk 3]
```diff
@@ -1,209 +1,177 @@
...
+- https://github.com/sendbird/Sendbird-JavaScript/tree/master/uikit-samples
+
+### Install UIKit for React with npm
+
+Enter the following on the command line with `npm` to install Sendbird UIKit.
+
+> **Note** : The [minimum requirements for UIKit](https://sendbird.com/docs/uikit/v1/javascript/getting-started/about-uikit#2-requirements) must be installed on your system to use `npm`.
+
+```bash
+npm install sendbird-uikit --save
```
Comment (sravan-s) [inline README.md:69]: Or using yarn
```bash
yarn add sendbird-uikit
```

https://github.com/sendbird/SendBird-Docs/pull/1807/files#diff-af3a2fc41e2ca46787b4519a4c00f717b8c525b3fc98759ace16571a568e9d77R36-R40

File: README.md [hunk 4]
```diff
@@ -1,209 +1,177 @@
...
+```
+
+To use the `App` component, add the following pattern:
+
+```javascript
+import { App as SendBirdApp } from "sendbird-uikit";
+import "sendbird-uikit/dist/index.css";
+
+const App = () => (
+        <Route id={'/chat'}>
```
Comment (sravan-s) [inline README.md:88]: nit - isnt 2 space tab better?

File: README.md (hunk 1 above)
Comment (sf-jeff-lim) [reply README.md:108]: Updated

File: README.md (hunk 2 above)
Comment (sf-jeff-lim) [reply README.md:49]: done!

File: README.md (hunk 3 above)
Comment (sf-jeff-lim) [reply README.md:72]: Done!

File: README.md (hunk 4 above)
Comment (sf-jeff-lim) [reply README.md]: Our standard is set as 4 spaces. Done!

**Pull request merged**